packages/lodestar/src/executionEngine/http.ts

```diff
@@ -50,7 +51,7 @@ export class ExecutionEngineHttp implements IExecutionEngine {
       rpc ??
       new JsonRpcHttpClient(opts.urls, {
         signal,
-        timeout: opts.timeout,
+        ...opts,
```

Comment:
maybe add some documentation in `ExecutionEngineHttp` class header regarding authentication port 8551